tests/unhappypath/unhappypath_test.go

```diff
@@ -0,0 +1,281 @@
+package tests
+
+import (
+	"fmt"
+	"math"
+	"os"
+	"strings"
+	"testing"
+	"time"
+
+	. "github.com/onsi/ginkgo"
+	"github.com/onsi/ginkgo/reporters"
+	. "github.com/onsi/gomega"
+	"github.com/portworx/sched-ops/k8s/stork"
+	"github.com/portworx/torpedo/drivers/node"
+	"github.com/portworx/torpedo/drivers/scheduler"
+	"github.com/portworx/torpedo/pkg/kvdbutils"
+	"github.com/portworx/torpedo/pkg/snapshotutils"
+	"github.com/portworx/torpedo/pkg/testrailuttils"
+	. "github.com/portworx/torpedo/tests"
+	"github.com/sirupsen/logrus"
+)
+
+var (
+	storkops = stork.Instance()
+)
+
+const (
+	dropPercentage      = 20
+	delayInMilliseconds = 250
+	//24 hours
+	totalTimeInHours              = 24
+	errorPersistTimeInMinutes     = 60 * time.Minute
+	snapshotScheduleRetryInterval = 10 * time.Second
+	snapshotScheduleRetryTimeout  = 5 * time.Minute
+)
+
+func TestBasic(t *testing.T) {
+	RegisterFailHandler(Fail)
+
+	var specReporters []Reporter
+	junitReporter := reporters.NewJUnitReporter("/testresults/junit_basic.xml")
+	specReporters = append(specReporters, junitReporter)
+	RunSpecsWithDefaultAndCustomReporters(t, "Torpedo : Basic", specReporters)
+}
+
+var _ = BeforeSuite(func() {
+	InitInstance()
+	if Inst().N.IsUsingSSH() {
+		for _, anode := range node.GetWorkerNodes() {
+			// TODO need to remove tcpdump installation using yum
+			// once https://portworx.atlassian.net/browse/PTX-12174 is resolved.
+			logrus.Infof("installing tcpdump on node %s", anode.Name)
+			cmd := "yum install -y tcpdump"
```

Comment:
This should be in spawn and not in torpedo.

Reply:
Removed this code